WordPress/gutenberg
# Editor: Fix block highlight rendering after block is moved

![Screen Capture on 2020-06-24 at 11-04-51](https://user-images.githubusercontent.com/2322354/85599930-fc81ee80-b61a-11ea-9954-530cc2cbb535.gif)

This update resolves the highlight rendering for blocks **after** they've been dragged and moved. The GIF demo above shows the highlight correctly disappearing after a block is moved, and another block is selected.

The solution involved removing the `requestAnimationFrame` wrapper that controlled the highlight state. It seemed like this was previously added to help with Toolbar rendering. I've done a bunch on my end. There does not appear to be any behavioural issues (or console errors) from removing this `rAF` wrapper.

## How has this been tested?

* Run `npm run dev`
* Add a block
* Add another block
* Drag the first block to re-order it. Highlight should appear.
* Drop the block
* Click on another block
* The highlight should disappear.


## Checklist:

Resolves: https://github.com/WordPress/gutenberg/issues/23413
